packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1101,7 +1101,14 @@ export default class Meeting extends StatelessWebexPlugin {
         this.selfId !== contentId &&
         disposition === FLOOR_ACTION.GRANTED
       ) {
-        newShareStatus = SHARE_STATUS.REMOTE_SHARE_ACTIVE;
+        if (this.mediaProperties.shareTrack.readyState === 'ended') {
+          newShareStatus = SHARE_STATUS.NO_SHARE;
```

Comment:
I don't believe you need to set `newShareStatus`. It should get set by `this.stopShare()`